sdks/go/types/common/types.go

```diff
@@ -484,6 +484,14 @@ func (b *BlockPayload) BuilderCommitment() (*BuilderCommitment, error) {
 	return &builderCommitment, nil
 }
 
+type PayloadQueryData struct {
+	Height 		 uint64 	   `json:"height"`
+	BlockHash    *TaggedBase64 `json:"block_hash"`
+	Hash 		 *TaggedBase64 `json:"hash"`
+	Size 		 uint64 	   `json:"size"`
+	BlockPayload *BlockPayload `json:"data"`
+}
+
```

Comment:
Nit: I just happended notice this typo "Sting", which wasn't introduced here, but could you update it as well? No worries if it's too much trouble.